packages/usa-file-input/src/test/file-input-accepts.spec.js

```diff
@@ -134,6 +138,38 @@ tests.forEach(({ name, selector: containerSelector }) => {
           true,
         );
       });
+
+      it("should display an error message", () => {
+        // add to our elements FileList
+        inputEl.files = fileList;
+        const e = new Event("change");
+        inputEl.dispatchEvent(e);
```

Comment:
I like this a lot! Since it uses the defined `inputEl` value, I made this an arrow function within in main tests rather than a function declaration outside of it. Let me know if that is the preferred approach!

Resolved in [31aae3c](https://github.com/uswds/uswds/pull/6168/commits/31aae3c383b5039f98e8867ca485fe99371239d8)